keyboards/handwired/macropad21/keymaps/via/rules.mk

```
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u4
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTLOADER = caterina
+COMBO_ENABLE = yes
+
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
+MOUSEKEY_ENABLE ?= yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes	# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no	# Console for debug(+400)
+COMMAND_ENABLE ?= yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE ?= no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE ?= no  # Enable keyboard backlight functionality
+AUDIO_ENABLE ?= no
+RGBLIGHT_ENABLE ?= no
```

Most of this isn't needed. In fact, only what has been changed from the default is needed. 

```suggestion
VIA_ENABLE = yes
```